README.md

```diff
@@ -37,48 +47,127 @@ const person = {
 }
 ```
 
-If you look closely, you can see that we've kind of seen this before, when we looped over an array containing objects. So it's not _that_ scary!
+If you look closely, you can see that we've kind of seen this before when we
+looped over an array containing objects. So it should be familiar!
+
+## Describe How to Access Inner Properties
+
+How would you imagine we'd access the `yearsHeld` field? 
+Let's try:
+
+``` javascript
+person.yearsHeld
+```
+
+We should see `undefined`. However, we can see that
+`yearsHeld` is a property of `occupation`, which in turn is a property of
+`person`.
+If we try ``occupation.yearsHeld`, that'll throw an error because `occupation`
+is not defined globally. It's only a property of `person`.
 
-How would you imagine we'd access the `yearsHeld` field? If we try `person.yearsHeld`, we get a big fat `undefined`. But we can see that `yearsHeld` is a property of `occupation`, which in turn is a property of `person`. So we could try `occupation.yearsHeld`, but that'll throw an error because `occupation` is not defined globally, only as a property of `person`. Hey, maybe there's a clue! What if we try `person.occupation`? We should see something like
+Let's try:
 
 ``` javascript
-{ title: "Senior Manager of Awesome", yearsHeld: 2 }
+person.occupation
 ```
 
-printed to console. Nice! So that suggests that if we do `person.occupation.yearsHeld` —
+We should see this printed to the console:
+
+``` javascript
+{ title: "District Manager", yearsHeld: 2 }
+```
+
+That suggests that if we type:
+
+```person.occupation.yearsHeld```
+
+We should now see the `yearsHeld` value:
 
 ``` javascript
 2
 ```
 
 Sweet!
 
-## Arrays in Arrays
+### Arrays in Arrays
+
+Let's get a little bit more abstract. In the above example, we had a name
```

Comment:
"Abstract" feels potentially confusing. Maybe "advanced" instead? Or complex? Abstract makes me think it's less accessible to understanding.